apps/perf-test/src/app/App.tsx

```diff
@@ -1,10 +1,9 @@
 import * as React from 'react';
-import { Dropdown, PrimaryButton, Stack, TextField, Text } from 'office-ui-fabric-react';
+import { Dropdown, PrimaryButton, Stack, TextField } from 'office-ui-fabric-react';
 import { Measurer, MeasurerTimings } from './Measurer';
-import { useTimer } from './useTimer';
 import { Scenarios } from './Scenarios';
 
-// tslint:disable
+/* eslint-disable react/jsx-no-bind */
```

Comment:
Why are we disabling this here? I don't see any binds.

Reply:
The new rule also covers lambdas and other ways of passing functions which will mutate every render.

Reply:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md